crazy_rl/test/test_numpy_and_jax.py

```diff
@@ -6,26 +6,26 @@
 from crazy_rl.multi_agent.numpy.surround.surround import Surround as NpSurround
 
 
-def compare(state, np_env, observations, terminations, truncations, rewards):
+def compare(state, jax_obs, jax_term, jax_trunc, jax_rewards, np_env, np_obs, np_term, np_trunc, np_rewards):
     assert (state.agents_locations[0] == np_env._agent_location["agent_0"]).all()
     assert (state.agents_locations[1] == np_env._agent_location["agent_1"]).all()
 
     assert state.timestep == np_env.timestep
 
-    assert (state.observations[0] == observations["agent_0"]).all()
-    assert (state.observations[1] == observations["agent_1"]).all()
+    assert (jax_obs[0] == np_obs["agent_0"]).all()
+    assert (jax_obs[1] == np_obs["agent_1"]).all()
 
-    assert state.terminations[0] == terminations["agent_0"]
-    assert state.terminations[1] == terminations["agent_1"]
+    assert jax_term[0] == np_term["agent_0"]
+    assert jax_term[1] == np_term["agent_1"]
 
-    assert state.truncations[0] == truncations["agent_0"]
-    assert state.truncations[1] == truncations["agent_1"]
+    assert jax_trunc[0] == np_trunc["agent_0"]
+    assert jax_trunc[1] == np_trunc["agent_1"]
 
-    assert state.rewards[0] > rewards["agent_0"] - 0.01
-    assert state.rewards[0] < rewards["agent_0"] + 0.01
+    assert jax_rewards[0] > np_rewards["agent_0"] - 0.01
```

Comment:
what does this test?

Reply:
It tests if the jax and numpy reward are the same, with an approximation

Reply:
Isn't something like `np.allclose(np.asarray(jax_rewards[0]), np_rewards["agent_0"]))` doable?